extensions-core/multi-stage-query/src/test/java/org/apache/druid/msq/test/DartComponentSupplier.java

```diff
@@ -116,6 +118,12 @@ final DruidNodeDiscoveryProvider getDiscoveryProvider()
       return null;
     }
 
+    @Provides
+    final DartSqlClients getDartClients()
+    {
+      return Mockito.mock(DartSqlClients.class);
```

Comment:
instead of adding a mock to the supplier ; could you try:
```
    @Provides
    final DruidNodeDiscoveryProvider getDiscoveryProvider()
    {
      return new CalciteTests.FakeDruidNodeDiscoveryProvider(Collections.emptyMap());
    }
```